### ISSUE TITLE
Current status of maintainer.

### ISSUE THREAD

**Issue by ericnewton76** [bcabanes/angular-image-cropper#63]
@bcabanes I see many issues not responded to in many months.

What's the status of any of these issues and pull requests?

**Comment by bcabanes** [comment]
Hi @ericnewton76,
I added a note on the readme, i have really not time anymore to maintain this library.

**Comment by ericnewton76** [comment]
Fair enough.  Thanks for the quick response.  I might hit you up for collab status and start merging pull requests possibly.
